src/parsing/index.ts

```diff
@@ -30,6 +30,7 @@ export interface ParseResult {
     suggestFunctions?: unknown;
     suggestGroupBys?: unknown;
     suggestIdentifiers?: IdentifierSuggestion[];
+    suggestSnippets?: boolean;
```

Comment:
It can't be false, so it's better to change it to 
```suggestion
    suggestSnippets?: true;
```

Same for `src/parsing/test/testing.ts`

Reply:
We can do it, but we can reserve `false` value for cases when we want to check that this key should not contain in actual result